src/main/interface/msp.c

```diff
@@ -2027,7 +2027,7 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         gpsSol.llh.lon = sbufReadU32(src);
         gpsSol.llh.alt = sbufReadU16(src) * 100; // alt changed from 1m to 0.01m per lsb since MSP API 1.39 by RTH. Received MSP altitudes in 1m per lsb have to upscaled.
         gpsSol.groundSpeed = sbufReadU16(src);
-        GPS_update |= 2;        // New data signalisation to GPS functions // FIXME Magic Numbers
```

Comment:
It would be nice to fix `FIXME Magic Numbers`

Reply:
I did not really understand what the previous dev meant with "Magic Numbers"?
I can introduce an `enum` or `const` values?

Reply:
Yes, https://en.m.wikipedia.org/wiki/Magic_number_(programming)

Reply:
`#define` or `enum` will do ...

Reply:
```
GPS_update |= GSP_MSP_UPDATE;
```